doc/release/release_0.20.rst

```diff
@@ -0,0 +1,479 @@
+Announcement: scikit-image 0.20.0
+=================================
+
+We're happy to announce the release of scikit-image v0.20.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+
+New features and improvements
+-----------------------------
+- Several footprint generating and consuming functions in ``skimage.morphology`` now support footprint decomposition.
+  By decomposing a footprint into several smaller ones, morphological operations can potentially be speed up.
+  The decomposed footprint can be generated with the new ``decomposition`` parameter of the functions ``rectangle``, ``diamond``, ``disk``, ``cube``, ``octahedron``, ``ball``, ``octagon`` in ``skimage.morphology``.
+  The ``footprint`` parameter of the functions ``binary_erosion``, ``binary_dilation``, ``binary_opening``, ``binary_closing``, ``erosion``, ``dilation``, ``opening``, ``closing``, ``white_tophat``, and ``black_tophat`` in ``skimage.morphology`` now accepts a tuple of a footprint and the number of times it should be applied iteratively.
+  (`#5482 <https://github.com/scikit-image/scikit-image/pull/5482>`_, `#6151 <https://github.com/scikit-image/scikit-image/pull/6151>`_)
+- Added support for the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` which performs better than the directed Hausdorff Distance (HD).
```

Comment:
What does "behave better" here?

Reply:
This refers to this statement in #5581

> Dubuisson et al. [1] demonstrated that the Modified Hausdorff Distance (MHD) outperforms the directed Hausdorff Distance (HD).

 From the [paper](https://doi.org/10.1109/ICPR.1994.576361)

> We have determined that, among the class of distance measures based on the Hausdorff distance, the
modified Hausdorff distance (MHD) is best for matching two objects based on their edge points. We have
shown that MHD has the following desirable properties: (i) its value increases monotonically as the
amount of difference between the two sets of edge points increases, and (ii) it is robust t o outlier points
that might result from segmentation errors.

My intention is that the notes answer "why did we add this" briefly. I'm trying to keep a good balance between not digging into every PR to deeply and still being clear. I'll try to clarify the advantages here.